Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -35,6 +37,26 @@ const GameContainer = styled.div`
   text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
 `;
 
+/**
+ * Wrapper to remove custom DOM attributes before rendering HTML DOM
+ * See: https://www.styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const BalloonSVGElement = ({ duration, isselected, ...props }:any) => (<BalloonSVG {...props} />);
+
+const DisplayBalloon = styled(BalloonSVGElement)`
+  width: 70px;
+  margin: 12px;
+  animation: ${(props: {duration: number}) => (props.duration === 0
+    ? undefined : css`${props.duration / 1000}s ${keyframes`${wobble}`} ease-in-out infinite;`)}
```

Comment:
Actually for this part I would prefer
```
  ${(props: {duration: number}) => props.duration !== 0
    && css`
      animation: ${props.duration / 1000}s ${keyframes`${wobble}`} ease-in-out infinite;
    `};
```
rather than setting animation to undefined?

Reply:
Aight! 👍 👍